src/frontend/scss/vendor/_icomoon.scss

```diff
@@ -0,0 +1,47 @@
+$icomoon-font-family: "icomoon" !default;
+$icomoon-font-path: "../fonts/icomoon" !default;
+
+$icon-file-text2: "\e926";
+
+@font-face {
+  font-family: '#{$icomoon-font-family}';
+  src:  url('#{$icomoon-font-path}/#{$icomoon-font-family}.eot?fn0yi1');
+  src:  url('#{$icomoon-font-path}/#{$icomoon-font-family}.eot?fn0yi1#iefix') format('embedded-opentype'),
+    url('#{$icomoon-font-path}/#{$icomoon-font-family}.ttf?fn0yi1') format('truetype'),
+    url('#{$icomoon-font-path}/#{$icomoon-font-family}.woff?fn0yi1') format('woff'),
```

Comment:
Can we add a `woff2` version? IMO all other formats besides `woff` and `woff2` are irrelevant but I think `woff2` will provide a lighter payload for 95% of users.

Reply:
unfortunately the `woff2` version is not available in the basic plan of icomoon